media-sound/pulseaudio/pulseaudio-14.2.ebuild

```diff
@@ -0,0 +1,318 @@
+		$(pa_meson_multilib_native_use_enable native-headset bluez5-native-headset)
+		$(pa_meson_multilib_native_use_enable ofono-headset bluez5-ofono-headset)
+		$(pa_meson_multilib_native_use_feature glib gsettings)
+		$(pa_meson_multilib_native_use_feature gtk)
+		$(pa_meson_multilib_native_use_feature jack)
+		-Dsamplerate=disabled # Matches upstream
+		$(pa_meson_multilib_native_use_feature lirc)
+		$(pa_meson_multilib_native_use_feature orc)
+		$(pa_meson_multilib_native_use_feature oss oss-output)
+		$(pa_meson_multilib_native_use_feature ssl openssl)
+		# tests involve random modules, so just do them for the native
+		$(pa_meson_multilib_native_use_enable test tests)
+		$(pa_meson_multilib_native_use_feature udev)
+		$(pa_meson_multilib_native_use_feature webrtc-aec)
+		$(pa_meson_multilib_native_use_feature zeroconf avahi)
+		$(pa_meson_multilib_native_use_feature equalizer fftw)
+		$(pa_meson_multilib_native_use_feature sox soxr)
+		-Ddatabase=$(multilib_native_usex gdbm gdbm simple) # TODO: tdb is also an option
+		$(meson_use gnome stream-restore-clear-old-devices) # TODO: Get ACK'ed on this
+		$(meson_feature glib)
+		$(meson_feature asyncns)
+		#$(meson_use cpu_flags_arm_neon neon-opt)
+		$(meson_feature tcpd tcpwrap) # TODO: system-wide specific?
+		$(meson_feature dbus)
+		$(meson_feature X x11)
+		$(meson_feature systemd)
+		$(meson_use ipv6)
+	)
+
+	if multilib_is_native_abi; then
+		# Make padsp work for non-native ABI, supposedly only possible with glibc; this is used by /usr/bin/padsp that comes from native build, thus we need this argument for native build
+		if use elibc_glibc; then
+			emesonargs+=( -Dpulsedsp-location="${EPREFIX}"'/usr/\\$$LIB/pulseaudio' )
+		fi
+	else
+		if ! use elibc_glibc; then
+			# Non-glibc multilib is probably non-existant but just in case:
+			ewarn "padsp wrapper for OSS emulation will only work with native ABI applications!"
+		fi
+	fi
+
+	meson_src_configure
+}
+
+multilib_src_compile() {
+	meson_src_compile
+
+	if multilib_is_native_abi; then
+		use doc && meson_src_compile doxygen
+	fi
+}
+
+multilib_src_test() {
+	# We avoid running the toplevel check target because that will run
+	# po/'s tests too, and they are broken. Officially, it should work
+	# with intltool 0.41, but that doesn't look like a stable release.
+	if multilib_is_native_abi; then
+		meson_src_test
+	fi
+}
+
+multilib_src_install() {
+	meson_src_install
+
+	if multilib_is_native_abi; then
+		if use doc ; then
+			# TODO: check it's installing into the right place
+			docinto html
+			dodoc -r doxygen/html/
+		fi
+	else
+		# remove foreign abi modules
+		rm -rf "${ED}"/usr/$(get_libdir)/pulse-*/
+	fi
+}
+
+multilib_src_install_all() {
+	if use system-wide; then
+		newconfd "${FILESDIR}"/pulseaudio.conf.d pulseaudio
+
+		use_define() {
+			local define=${2:-$(echo $1 | tr '[:lower:]' '[:upper:]')}
+
+			use "$1" && echo "-D$define" || echo "-U$define"
+		}
+
+		unifdef $(use_define zeroconf AVAHI) \
```

Comment:
I'll assume you mean that unifdef as a whole count as an external command rather than the echo which is a shell built-in (since ebuilds are explicitly written in bash, echo would never be an external command, IMO), and I'll commit a fix for that. If I made a mistake, please do explain in greater detail.